src/PKSim.Presentation/Presenters/Main/MenuAndToolBarPresenter.cs

```diff
@@ -510,18 +516,24 @@ public void Visit(SensitivityAnalysis sensitivityAnalysis)
 
       public void Visit(ParameterIdentification parameterIdentification)
       {
+         _currentlyActivePI = parameterIdentification.Id;
+         _parameterIdentificationRunning = _activePIs.Contains(parameterIdentification.Id);
          updateParameterIdentificationItems(parameterIdentification);
+         _eventPublisher.PublishEvent(new ParameterIdentificationSelectedEvent(parameterIdentification));
       }
 
       public void Handle(ParameterIdentificationStartedEvent parameterIdentificationEvent)
       {
+         _activePIs.Add(parameterIdentificationEvent.ParameterIdentification.Id);
          _parameterIdentificationRunning = true;
          updateParameterIdentificationItems(parameterIdentificationEvent.ParameterIdentification);
```

Comment:
this flag is also `_parameterIdentificationRunning `  suprirsing. I think this could be a parameter of the updateParameterIdentificationItems no?
You do not need to tract activePI IMO here as well. Let's discuss

Reply:
Sure, we can discuss. Not sure if you are working now. Otherwise we talk tomorrow.